source/D2Game/src/GAME/Level.cpp

```diff
@@ -292,7 +292,7 @@ void __fastcall LEVEL_ChangeAct(D2GameStrc* pGame, D2ClientStrc* pClient, int32_
     }
 
     CLIENTS_SetRoomInClient(pClient, nullptr);
-    D2GAME_PACKETS_SendPacket0x4F_StartMercList_6FC3C6F0(pClient, 5u);
+    D2GAME_PACKETS_SendPacket0x4F_UpdateClientState(pClient, 5u);
```

Comment:
Could use enum?

Reply:
I'd like to confirm that it is indeed the same enum that is being used here. Because for example `GAME_SendGameInit` Sends the packet with `0` just before the call to `CLIENTS_SetClientState(pClient, 1u);`, and then sends the packet with `2` at the end so... `GAME_EndGame` also sends the packet `5` (I mean, why not, since the player is still connected but in the process of being kicked out), and then `6`.